assets/css/app.css

```
@@ -1,7 +1,58 @@
-@tailwind base;
-@tailwind components;
-@tailwind utilities;
+@import "tailwindcss";
 
+@config "../tailwind.config.js";
+
+@plugin "@tailwindcss/typography";
+
+@plugin "daisyui" {
```

Stylelint (17.9.0)

[error] Unexpected unknown at-rule "@plugin" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)